Cannot compile Blazor Server application using basic theme

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Created a new Blazor Server application selecting the basic theme but when I compile I get the following error:

C:\Projects\Acme.BookStore\src\Acme.BookStore.Blazor\Pages\_Host.cshtml(4,42): error CS0234: The type or namespace name 'LeptonXLiteTheme' does not exist in the namespace 'Volo.Abp.AspNetCore.Components.Web' (are you missing an assembly reference?) [C:\Projects\Acme.BookStore\src\Acme.BookStore.Blazor\Acme.BookStore.Blazor.csproj]

Build FAILED.

I tried removing the following from ..\src\Acme.BookStore.Blazor\Pages\_Host.cshtml

@using Volo.Abp.AspNetCore.Components.Web.LeptonXLiteTheme.Themes.LeptonXLite

but now get the following error:

C:\Projects\Acme.BookStore\src\Acme.BookStore.Blazor\Pages\_Host.cshtml(23,81): error CS0246: The type or namespace name 'App' could not be found (are you missing a using directive or an assembly reference?) [C:\Projects\Acme. BookStore\src\Acme.BookStore.Blazor\Acme.BookStore.Blazor.csproj]



### Reproduction Steps

abp new Acme.BookStore -u blazor-server --theme basic

dotnet build

### Expected behavior

_No response_

### Actual behavior

_No response_

### Regression?

_No response_

### Known Workarounds

_No response_

### Version

7.1.1

### User Interface

Blazor Server

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

### Other information

_No response_

--- a/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Building/Steps/ChangeThemeStep.cs
+++ b/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Building/Steps/ChangeThemeStep.cs
@@ -844,6 +844,13 @@ public class ChangeThemeStep : ProjectBuildPipelineStep
                 $"Blazor{defaultThemeName}ThemeBundles.Scripts.Global",
                 "BlazorBasicThemeBundles.Scripts.Global"
             );
+
+            ChangeNamespace(
+                context,
+                $"/MyCompanyName.MyProjectName.{project.Key}/Pages/_Host.cshtml",
+                $"Volo.Abp.AspNetCore.Components.Web.{defaultThemeName}Theme.Themes.{defaultThemeName}",
+                "Volo.Abp.AspNetCore.Components.Web.BasicTheme.Themes.Basic"
+            );
         }
     }
 

